examples/asr/conf/speech_to_text_hf_finetune.yaml

```diff
@@ -0,0 +1,172 @@
+name: "Speech_To_Text_HF_Finetuning"
+
+# use `init_from_nemo_model` or `init_from_pretrained_model` to initialize the model
+# We do not currently support `init_from_ptl_ckpt` to create a single script for all types of models.
+init_from_nemo_model: null # path to nemo model
+init_from_pretrained_model: null  # name of pretrained NeMo model, e.g., `stt_en_fastconformer_transducer_large`
+
+model:
+  sample_rate: 16000
+  compute_eval_loss: false # eval samples can be very long and exhaust memory. Disable computation of transducer loss during validation/testing with this flag.
+  log_prediction: true # enables logging sample predictions in the output during training
+  rnnt_reduction: 'mean_volume'
+  skip_nan_grad: false
+
+  # configs for huggingface load_dataset function
+  data_path: "librispeech_asr"
+  data_name: null
+  num_proc: 8  # Number of processes when downloading and generating the dataset locally.
+  streaming: false # set True to use streaming mode, which doesn't wait for data downloading but each training step takes longer in the first epoch. If True, you'll need to specify trainer.max_steps instead of trainer.max_epochs.
+
+  # keys for audio, sample_rate and text in the huggingface dataset, keys seperated by `.` for nested fields.
+  # An example of data item in the `librispeech_asr` dataset:
+  #   {'chapter_id': 141231,
+  #  'file': '/home/patrick/.cache/huggingface/datasets/downloads/extracted/b7ded9969e09942ab65313e691e6fc2e12066192ee8527e21d634aca128afbe2/dev_clean/1272/141231/1272-141231-0000.flac',
+  #   'audio': {'path': '/home/patrick/.cache/huggingface/datasets/downloads/extracted/b7ded9969e09942ab65313e691e6fc2e12066192ee8527e21d634aca128afbe2/dev_clean/1272/141231/1272-141231-0000.flac',
+  #   'array': array([-0.00048828, -0.00018311, -0.00137329, ...,  0.00079346,
+  #           0.00091553,  0.00085449], dtype=float32),
+  #   'sampling_rate': 16000},
+  #  'id': '1272-141231-0000',
+  #  'speaker_id': 1272,
+  #  'text': 'A MAN SAID TO THE UNIVERSE SIR I EXIST'}
+  audio_key: "audio.array"
+  sample_rate_key: "audio.sampling_rate"
+  text_key: "text"
+
+  # simple text cleaning, by default converts all chars to lower-case and only keeps alpha-numeric chars.
+  normalize_text: true
+  symbols_to_keep: "-'"
```

Comment:
why `-`? I believe for our current models we dont have `-` only `'`

Reply:
updated to only `'`